src/5.0_to_5.1/calculated-measures/migrateCalculatedMeasuresInDashboards.test.ts

```diff
@@ -1,8 +1,8 @@
 import { uiDashboardsFolder } from "../__test_resources__/uiDashboardsFolder";
 import { migrateCalculatedMeasuresInDashboards } from "./migrateCalculatedMeasuresInDashboards";
-import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { sandboxDataModel } from "@activeviam/data-model-5.1/dist/__test_resources__";
```

Comment:
Do we need to get it from `dist/__test_resources__`?

Reply:
It's the standard that we started doing recently in the monorepo, to avoid having it end up in the application bundle.

Reply:
Sweet. Looks good to me in that case. I can't approve as I'm the author, but I'll leave a checkmark here. :heavy_check_mark: